src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -817,7 +817,7 @@ class IrsPlan extends React.Component<
   }
   /** onSaveEditNameButtonClick - handler disabling inline-editing */
   private onSaveEditNameButtonClick(e: MouseEvent) {
-    e.preventDefault();
+    preventDefault();
```

Comment:
We'll need to pass `e` into `preventDefault` here

Reply:
oh yeah. :+1: